.github/workflows/docker.yml

```diff
@@ -0,0 +1,110 @@
+name: docker
+
+on:
+    push:
+        branches:
+            - unstable
+            - stable
+
+env:
+    DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+    DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+    IMAGE_NAME: sigp/lighthouse
+
+jobs:
+    extract-branch-name:
+        runs-on: ubuntu-18.04
+        steps:
+            - name: Extract branch name
+              run: echo "::set-output name=BRANCH_NAME::$(echo ${GITHUB_REF#refs/heads/})"
+              id: extract_branch
+        outputs:
+            BRANCH_NAME: ${{ steps.extract_branch.outputs.BRANCH_NAME }}
+    build-docker-arm64:
+        runs-on: ubuntu-18.04
+        needs: [extract-branch-name]
+        steps:
+            - uses: actions/checkout@v1
+            - name: Dockerhub login
+              run: |
+                  echo "${DOCKER_PASSWORD}" | docker login --username ${DOCKER_USERNAME} --password-stdin
+            - uses: actions-rs/toolchain@v1
+              with:
+                  toolchain: stable
+            - name: Cross build lighthouse binary
+              uses: actions-rs/cargo@v1
+              with:
+                  use-cross: true
+                  command: build
+                  args: --release --manifest-path lighthouse/Cargo.toml --target aarch64-unknown-linux-gnu --features portable
+            - name: Cross build lcli binary
+              uses: actions-rs/cargo@v1
+              with:
+                  use-cross: true
+                  command: build
+                  args: --release --manifest-path lcli/Cargo.toml --target aarch64-unknown-linux-gnu --features portable
+            - name: Move cross-built ARM binaries into Docker scope
+              run: |
+                  mkdir ./bin;
+                  mv ./target/aarch64-unknown-linux-gnu/release/lighthouse ./bin;
+                  mv ./target/aarch64-unknown-linux-gnu/release/lcli ./bin;
+            - name: Build Docker Buildx
+              run: |
+                  export DOCKER_BUILDKIT=1;
+                  docker build --platform=local -o . git://github.com/docker/buildx;
+                  mkdir -p ~/.docker/cli-plugins;
+                  mv buildx ~/.docker/cli-plugins/docker-buildx;
+            - name: Create Docker Builder
+              run: |
+                  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes;
```

Comment:
(oops, forgot to submit this comment)

I'm confused about what this does. It seems to install some binaries or set up some configuration so that docker can run ARM images, but we don't make use of that functionality do we? Or does `docker buildx build --platform=linux/arm64` require it?

Reply:
I think we could get away with not using it if we didn't have a `RUN` command in `Dockerfile.cross`. But because of that, we have to run the ARM64 version of the debian image during the build

Reference comment from a buildx issue: https://github.com/docker/buildx/issues/132#issuecomment-522177152